boa_parser/src/parser/cursor/mod.rs

```diff
@@ -75,7 +72,7 @@ where
     }
 
     /// Advances the cursor and returns the next token.
-    #[inline]
+
```

Comment:
Don't like the empty line between the comment and function
There's a few more lower in the file